include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.td

```diff
@@ -363,6 +363,16 @@ def FInstanceLike : OpInterface<"FInstanceLike", [
   let description = "Provide common  instance information.";
 
   let methods = [
+    InterfaceMethod<"Get the name of the instantiated module",
+    "::llvm::StringRef", "getReferencedModuleName", (ins),
```

Comment:
Just curious, how will these InstanceLike methods work for instances that have multiple options? What will be returned by these methods?

Reply:
(Oh, are the choice versions not `InstanceLike`? Okay)

Reply:
I will be pushing these methods further down the hierarchy.